currency-rates/arch-go.yaml

```diff
@@ -0,0 +1,53 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 75
+dependenciesRules:
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.clients.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.handlers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.repo.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.routers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.handlers.**"
+        - "**.tools.**"
+  - package: "**.scheduler.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.tools.**"
+    shouldOnlyDependsOn:
+      internal:    
+  - package: "**.tests.**"
+    shouldOnlyDependsOn:
+      internal:    
+  - package: "**.cmd.**"
+    shouldOnlyDependsOn:
+      internal:      
+        - "**.app.**"
+        - "**.tools.**"
+
+functionsRules:
```

Comment:
What does this responsible for?

Reply:
Didn't get your question

Reply:
Explain me more about functionsRules please.  
For what does "maxLines" responsible for?

Reply:
These are rules for functions. It's more like lint functions. Not related to architecture, I just wanted to try this part of the new lib's functionality ))